enclave_manager/avalon_enclave_manager/singleton/enclave_manager.py

```diff
@@ -609,8 +609,9 @@ def main(args=None):
     import utility.logger as plogger
 
     # parse out the configuration file first
-    conffiles = ["tcs_config.toml"]
-    confpaths = [".", TCFHOME + "/" + "config"]
+    conffiles = ["config.toml"]
```

Comment:
rename the config.toml to something like singleton_config.toml

Reply:
I deliberately named them to config.toml @pankajgoyal2  as they are within their corresponding directories. Prepending the directory name again would make it verbose I reckon.

Reply:
It might look verbose but it would be more clear. Same way has been followed in other avalon components.

Reply:
Done @pankajgoyal2. Though I still feel that this should not be so elsewhere as well. Can ponder upon it later. :|